pom.xml

```diff
@@ -168,6 +168,7 @@
   </profiles>
   <properties>
     <jackson.version>2.13.4</jackson.version>
```

Comment:
Any issue with bumping this to `2.14.0`?

Reply:
https://github.com/twilio/twilio-java/pull/721